### ISSUE TITLE
【论文复现营】x2paddle转换时torch.optim.Optimizer和paddle.optimizer.Optimizer接口无法对应

### ISSUE THREAD

**Issue by Felix-python** [PaddlePaddle/X2Paddle#583]
复现代码时需要用到pytorch-image-models/timm/scheduler中的代码，使用x2paddle转换失败，发现torch.optim.Optimizer和paddle.optimizer.Optimizer接口无法对应，人工修改后仍存在问题，paddle的Optimizer类中缺少param_groups属性
转换失败代码的链接：https://github.com/rwightman/pytorch-image-models/tree/master/timm/scheduler

**Comment by SunAhong1993** [comment]
paddle的Optimizer类与PyTorch的使用机制较为不一致，所以属性也是不一致的，param_groups属性可以通过重写下Optimizer这个API自行添加。
